x/mint/keeper/keeper.go

```diff
@@ -121,6 +115,33 @@ func (k Keeper) SetParams(ctx sdk.Context, params types.Params) {
 	k.paramSpace.SetParamSet(ctx, &params)
 }
 
+// GetTruncationDelta returns the truncation delta.
+// Only developer vesting and mint module accounts are allowed.
+// Returns error if invalid module account. Panics if key is invalid.
+func (k Keeper) GetTruncationDelta(ctx sdk.Context, moduleAccountName string) (sdk.Dec, error) {
+	storeKey, err := getTruncationStoreKeyFromModuleAccount(moduleAccountName)
+	if err != nil {
+		return sdk.Dec{}, err
+	}
+	return osmoutils.MustGetDec(ctx.KVStore(k.storeKey), storeKey), nil
+}
+
+// SetTruncationDelta sets the truncation delta, returning an error if any. nil otherwise.
+// Only developer vesting and mint module accounts are allowed.
+// Returns error if invalid module account.
+func (k Keeper) SetTruncationDelta(ctx sdk.Context, moduleAccountName string, truncationDelta sdk.Dec) error {
+	storeKey, err := getTruncationStoreKeyFromModuleAccount(moduleAccountName)
+	if err != nil {
+		return err
+	}
+
+	if truncationDelta.LT(sdk.ZeroDec()) {
+		return sdkerrors.Wrapf(types.ErrInvalidAmount, "truncation delta must be positive, was (%s)", truncationDelta)
+	}
```

Comment:
nit: we should probably check trucationDelta first before fetching the store key to prevent unnecessary logic from executing.

Reply:
eh, I would not be concerned about performance of this. This is not the bottleneck, its once per epoch behavior.